src/util.c

```diff
@@ -559,6 +559,17 @@ cc_oci_enable_networking (void)
 	return enable;
 }
 
+/*!
+ * Convert the value stored in buffer to little endian
+ *
+ * \param buf Buffer storing the big endian value
+ *
+ * \return \c guint32 in big endian order.
+ */
+guint32 cc_oci_get_big_endian_32(const char *buf) {
+	return (guint32)(buf[0] >> 24 | buf[1] >> 16 | buf[2] >> 8 | buf[3]);
```

Comment:
we got a coverity issue with this on previous commit related to #362 

> 
> CID 1375208 (#3-1 of 3): Bad bit shift operation (BAD_SHIFT)
> 1. large_shift: In expression buf[2] >> 8, right shifting buf[2] by more than 7 bits always yields either all zeros or all ones. The shift amount is 8.

` return (uint32_t)(buf[0] >> 24 | buf[1] >> 16 | buf[2] >> 8 | buf[3]);
`